pages/_next-prefetcher.js

```diff
@@ -0,0 +1,108 @@
+/* global self */
```

Comment:
I tried at first, but it adds more code (because I use async await). But writing it with pure ES5 make it pretty simple (I just want this to work).
But I'd like to try something different now.

This is the all the features we need to add to the service worker code. Writing ES5 makes it smaller. Do we need to use ES6 and transpile it?